### PR TITLE
Update update-tool.md

### PR DESCRIPTION
**Pull request by bcblackmer**
Readability updates.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by chamindi-a — APPROVED** [review]

**Review by chamindi-a — CHANGES_REQUESTED** [review]

File: en/docs/updates/update-tool.md [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
+    -   Before you run the Update Tool, **stop the product pack from running**.
+    -   If you have symbolic links (also known as symlinks) in the product pack, back up the original files as they could be modified during the update.
+    -   The updates commands should be performed in the command prompt under the folder path of <strong>`<product-home>/bin`</strong>
+    -   You may need to run the relevant update command twice. The first time you execute `./wso2update_<os>`, the Update Tool will update itself. The second time you execute `./wso2update_<os>` will update the product pack. 
 
 !!! important
-    We have compiled the Update Tool to be compatible with AMD64 (X86_64) architecture, and we have three separate binaries for Windows, Linux and MacOS.<br>
-    If your runtime environment doesn't belong to any mentioned setup above, <strong>we recommend you to keep a builder machine of the above mentioned kind, run the Update Tool in the builder machine, and ship the updated packs to your runtime environment.</strong>
+    The Update Tool is be compatible with AMD64 (X86_64) architecture, and there are three separate binaries for Windows, Linux, and MacOS.<br>
```
[the remark this answers is not in the document]
Comment (bcblackmer) [reply en/docs/updates/update-tool.md:12]: Sounds fine. Does this mean you're removing the references to the windows, linux, and macos binaries?

**Review by chamindi-a — APPROVED** [review]

File: en/docs/updates/update-tool.md [hunk 2]
```diff
@@ -1,17 +1,16 @@
 ## What is the Update Tool?
 
-Update tool is a CLI distribution that is separate for Linux, MacOS and Windows distributions. Bringing your product up to date 
-with the Update Tool is very easy.It is as simple as running one of the commands that matches with your Operating System.
+The Update Tool is a CLI distribution for Linux, MacOS and Windows distributions that streamlines updating WSO2 products. The Update Tool will automatically download the latest updates for your WSO2 products and install them. It will also take a backup of your current installation before updating it.
```
Comment (jsdjayanga) [inline en/docs/updates/update-tool.md:3]: We cant say "The Update Tool will automatically download the latest updates", the user has to run it

File: en/docs/updates/update-tool.md [hunk 3]
```diff
@@ -1,17 +1,16 @@
...
 
 !!! tip "Before you begin"
-    -   Before you run the update tool, **remember to stop** the product pack from running.
-    -   If you have symbolic link(s) (also known as symlink(s)) in the product pack, back up the original file(s) as they could be modified during the update.
-    -   The following commands should be performed in command prompt under the folder path of <strong>`<product-home>/bin`</strong>
-    -   At first you may need to run the relevant update command twice. First time you execute `./wso2update_<os>` updates the Update Client Tool. The subsequent `./wso2update_<os>` updates the product pack. 
+    -   Before you run the Update Tool, **stop the product pack from running**.
+    -   If you have symbolic links (also known as symlinks) in the product pack, back up the original files as they could be modified during the update.
+    -   The updates commands should be performed in the command prompt under the folder path of <strong>`<product-home>/bin`</strong>
```
Comment (jsdjayanga) [inline en/docs/updates/update-tool.md:8]: Command prompt -> terminal

File: en/docs/updates/update-tool.md (hunk 2 above)
Comment (bcblackmer) [reply en/docs/updates/update-tool.md]: Cool; changed.

**Review by jsdjayanga — APPROVED** [review]

**Review by jsdjayanga — APPROVED** [review]

**Pull request merged**